module/data.js

```diff
@@ -0,0 +1,33 @@
+const speaker = [
+
+  {
+    id: 'speaker1',
+    image: './images/four.jpg',
+    name: 'Herbart obi',
+    profession: 'Executive director of the Wikimedia Foundation',
+    aboutMe: 'Benkler studies commons-based peer production.',
+  },
+  {
+    id: 'speaker2',
+    image: './images/IMG_2508.jpg',
+    name: 'Bright Eze',
+    profession: 'Berkman Professor of Enterpreneurial',
+    aboutMe: 'Benkler studies commons-based peer production.',
+  },
+  {
+    id: 'speaker3',
+    image: './images/two.jpg',
+    name: 'Alima Buki',
+    profession: 'Executive director of the Wikimedia Foundation',
+    aboutMe: 'Benkler studies commons-based peer production.',
+  },
+  {
+    id: 'speaker4',
+    image: './images/IMG_6177 copy.jpg',
+    name: 'Progress Eze',
+    profession: 'Berkman Professor of Enterpreneurial',
+    aboutMe: 'Benkler studies commons-based peer production.',
+  },
+];
+
```

Comment:
- [ ] Kindly make sure you match the speaker's design on the Mobile version. By default, the user can only see 2 speakers but once clicked the `See More` button he can see the rest of the speakers.

| Template design | Your design |
| ------------------- | --------------------- |
| ![Screenshot from 2023-01-05 20-46-57](https://user-images.githubusercontent.com/83042742/212492816-2f22400b-2bbc-4a68-a854-60659a38ea15.png) | ![Screenshot from 2023-01-14 21-31-44](https://user-images.githubusercontent.com/83042742/212492861-3960ee81-56d2-4828-98c6-18da8798bb68.png) |

Reply:
- [ ] [REQUIRED] I would like to strongly recommend you implement this as required by the previous reviewer in order to make sure your design matches the original design speaker's design on the Mobile version. By default, the user can only see 2 speakers but once clicked the See More button he can see the rest of the speakers.Please fix it :+1: